modules/core/src/test/java/org/apache/ignite/internal/processors/cluster/ChangeTopologyWatcherTest.java

```diff
@@ -344,4 +345,43 @@ public void testBaselineAutoAdjustIgnoreClientNodes() throws Exception {
 
         assertTrue(isCurrentBaselineFromOneNode(ignite0));
     }
+
+    /**
+     * @throws Exception if failed.
+     */
+    @Test
+    public void testBaselineAutoAdjustDisableByDefaultBecauseNotNewCluster() throws Exception {
+        System.setProperty(IGNITE_BASELINE_AUTO_ADJUST_ENABLED, "false");
+        try {
+            Ignite ignite0 = startGrids(2);
+
+            ignite0.cluster().active(true);
+
+            ignite0.cluster().baselineAutoAdjustTimeout(0);
+
+            stopAllGrids();
+        }
+        finally {
+            System.clearProperty(IGNITE_BASELINE_AUTO_ADJUST_ENABLED);
+        }
+
+        Ignite ignite0 = startGrids(2);
+
+        ignite0.cluster().active(true);
+
+        Set<Object> initBaseline = ignite0.cluster().currentBaselineTopology().stream()
```

Comment:
I think that it would be much easier to read if you compared size of baseline nodes set, not their ids. Is that ok?